noxfile.py

```diff
@@ -386,7 +386,10 @@ def test_plugins_in_directory(
     install_hydra(session, install_cmd)
     selected_plugin = select_plugins(session=session, directory=directory)
     for plugin in selected_plugin:
-        cmd = list(install_cmd) + [os.path.join(directory, plugin.path)]
+        cmd = list(install_cmd) + [
+            os.path.join(directory, plugin.path),
+            "--use-feature=in-tree-build",  # https://github.com/pypa/pip/issues/7555
+        ]
```

Comment:
remove. This will become the default at some point and the error has nothing to do with it.

Reply:
removed.